### PR TITLE
added email for lisa zapson

### PR DESCRIPTION
**Pull request by lzapson**
## Description of change
add email to settings.yml to be flipper admin

### CODE REVIEW

File: config/settings.yml [hunk 1]
```diff
@@ -223,15 +222,15 @@ modules_appeals_api:
...
+      every: ["24h", first_in: "45m"]
     appeals_api_hlr_status_batch:
       class: AppealsApi::HigherLevelReviewUploadStatusBatch
       description: "Update HigherLevelReview statuses with their Central Mail status"
       every: "60m"
     appeals_api_nod_cleanup_pii:
       class: AppealsApi::NoticeOfDisagreementCleanUpWeekOldPii
       description: "Remove PII of NoticeOfDisagreements"
-      every: ['24h', first_in: '45m']
+      every: ["24h", first_in: "45m"]
```
Comment (ericboehs) [inline config/settings.yml:233]: These quotes and formatting changes don't seem related to the PR. Could you remove these please?

**Comment by lzapson** [comment]
several format changes were added by mistake.
